license_header

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
```

Comment:
should this read "This file is part of Parity sol-rs"? also wherever "Parity" is mentioned below